src/game_api/screen_arena.hpp

```diff
@@ -127,53 +91,24 @@ class ScreenArenaStagesSelect : public Screen // ID: 22 and 24
     TextureRenderingInfo players_turn_scroll;
     TextureRenderingInfo players_turn_scroll_handle;
     TextureRenderingInfo grid_player_icon;
-    float unknown30;
-    float unknown31;
-    float unknown32;
-    uint32_t unknown33;
-    uint32_t unknown34;
-    uint32_t unknown35;
-    uint32_t unknown36;
-    uint32_t unknown37;
-    uint32_t unknown38;
-    uint32_t unknown39;
-    uint32_t unknown40;
-    uint32_t unknown41;
-    uint32_t unknown42;
-    uint32_t unknown43;
-    uint32_t unknown44;
-    uint32_t unknown45;
-    uint32_t unknown46;
-    uint32_t unknown47;
+    float unknown30;                                         // padding probably
+    custom_unordered_map<uint32_t, float> stages_to_gay_out; // key is the stage, float is how gray out it is
```

Comment:
gray

Reply:
How did I make it correct in plugin but not here 🤦